pkgs/applications/editors/gnome-builder/python-libprefix.patch

```diff
@@ -0,0 +1,41 @@
+--- gnome-builder-3.28.4/meson_options.txt.orig	2018-07-29 15:36:03.605081188 -0700
++++ gnome-builder-3.28.4/meson_options.txt	2018-07-29 15:36:19.877608836 -0700
+@@ -15,6 +15,8 @@
+ option('with_help', type: 'boolean')
+ option('with_docs', type: 'boolean', value: false)
+ 
++option('python_libprefix', type: 'string')
++
+ # Plugins
+ # Ideally we want many of these to be defined in the plugin dir:
+ #   https://github.com/mesonbuild/meson/issues/707
+--- gnome-builder-3.28.4/src/libide/meson.build.orig	2018-07-29 15:36:03.605081188 -0700
++++ gnome-builder-3.28.4/src/libide/meson.build	2018-07-29 15:36:43.237365710 -0700
+@@ -161,6 +161,10 @@
+ endif
+ 
+ # We want to find the subdirectory to install our override into:
++python_libprefix = get_option('python_libprefix')
++if python_libprefix != ''
++  pygobject_override_dir = join_paths(get_option('libdir'), python_libprefix, 'site_packages', 'gi', 'overrides')
```

Comment:
`site_packages` should contain dash, not underscore.

Reply:
Oh. I guess the only plugin that actually needed `gi/overrides/Ide.py` was the `jedi_plugin`, and I ignored the error message from that because everything else was working fine. But getting the path correct fixes that error.